resources/css/breadcrumb.css

```diff
@@ -14,4 +14,5 @@
 
 .navpath > * {
   display: inline-block;
+  @apply leading-4;
```

Comment:
For consistency, apply the game page's line-height change site wide. This greatly improves the readability of some forum thread breadcrumbs that contain emojis, such as AOTW.